docs/DeveloperGuide.md

```diff
@@ -355,62 +355,108 @@ We have chosen to implement the `delete` command to accept the index of the pers
 
 #### **Find Feature** : `find`
 
-The `find` command lets users search for persons by substring matching. The user can select any parameter to search under: `NAME`, `EMAIL`, `TAG`, `MATRIC`, `REFLECTION`, `STUDIO`, and `TAGS` can all be used. E.g. to search for all persons under studio `S2`, the user can use `find s|s2`.
+The `find` command lets users search for persons by substring matching. The user can select any parameter to search under: 
+`NAME`, `EMAIL`, `TAG`, `MATRIC`, `REFLECTION`, `STUDIO`, and `TAGS` can all be used. E.g. to search for all persons under studio `S2`, the user can use `find s|s2`.
```

Comment:
typo for `s2` since studio has to start with a capital "S" so maybe want to change to `find s|S2`?

Reply:
this still works since the search is not case sensitive